apps/comments/src/components/Comment.vue

```diff
@@ -105,12 +105,12 @@ import NcActions from '@nextcloud/vue/dist/Components/NcActions'
 import NcActionSeparator from '@nextcloud/vue/dist/Components/NcActionSeparator'
 import NcAvatar from '@nextcloud/vue/dist/Components/NcAvatar'
 import NcButton from '@nextcloud/vue/dist/Components/NcButton'
-import NcRichContenteditable from '@nextcloud/vue/dist/Components/NcRichContenteditable'
 import RichEditorMixin from '@nextcloud/vue/dist/Mixins/richEditor'
 import ArrowRight from 'vue-material-design-icons/ArrowRight'
 
 import Moment from './Moment'
 import CommentMixin from '../mixins/CommentMixin'
+const NcRichContenteditable = () => import('@nextcloud/vue/dist/Components/NcRichContenteditable')
```

Comment:
Is this something we could fix in our libraries?

Reply:
[Async components](https://v2.vuejs.org/v2/guide/components-dynamic-async.html) is a Vue feature. There's no bug, so there's nothing to "fix" as such.

Reply:
Yeah i more meant if we can help all users of our libraries to do that out of the box, instead of having to bring it manually to all apps and points

Reply:
No. Whether a component can be lazy loaded depends on the assumptions the caller is making. For example, if some other function has some side effects that affects the component, lazy loading will break things. This is as out-of-the-box as it can get.

Reply:
One thing worth figuring out would be the reason `@nextcloud/vue` is so bloated (100KB per component at times). If that can be fixed then we don't need lazy loading at all.

Reply:
The duplication that is in`@nextcloud/vue` is getting addressed in #36383

Reply:
```suggestion
const NcRichContenteditable = () => import(/* webpackPreload: true */ '@nextcloud/vue/dist/Components/NcRichContenteditable')
```

see https://webpack.js.org/guides/code-splitting/#prefetchingpreloading-modules